src/OOVPADatabase/Xapi/4831.inl

```diff
@@ -206,6 +206,32 @@ OOVPA_SIG_MATCH(
     //
 );
 
+// ******************************************************************
+// * XInputSetLightgunCalibration
+// ******************************************************************
+OOVPA_SIG_HEADER_XREF(XInputSetLightgunCalibration,
+                      4831,
+
+                      XRefNoSaveIndex,
+                      XRefOne)
+OOVPA_SIG_MATCH(
+
+    XREF_ENTRY(0x1C, XREF_KT_FUNC_KeRaiseIrqlToDpcLevel),
```

Comment:
Can you add a comment what exactly this entry do? Call? Jump to?

Also, perhaps add the instruction's bytes before this entry for safeguard purpose? (just a nit)